root/etc/e-smith/templates/etc/cron.d/nethserver-blacklist/10base

```diff
@@ -10,7 +10,7 @@
 
     my $dns_status = $ftl{'status'} || 'disabled';
     if ($dns_status eq 'enabled') {
-        $OUT .= '*/20 * * * * root sleep $(( ( RANDOM \% 60 ) )); /usr/share/nethserver-blacklist/download dnss'
+        $OUT .= '0 23 * * * root sleep $(( ( RANDOM \% 60 ) )); /usr/share/nethserver-blacklist/download dnss'
```

Comment:
I propose also to spread a bit more requests to the servers.
Suggestion: increase random delay from 1 to 15 minutes
```suggestion
        $OUT .= '0 23 * * * root sleep $(( ( RANDOM \% 900 ) )); /usr/share/nethserver-blacklist/download dnss'
```